### ISSUE TITLE
/f not run

### ISSUE THREAD

**Issue by SayR0** [drtshock/Factions#638]
Hello, I use your FactionUuid plugin but I have a problem. Indeed, in the config.yml I edit the / f help for a custom help so I do not use the old / f help. However now when we do / f this command not run but  / f help run. The problem is when one is mistaken command (in game) it display you got the wrong order and to do / f exept that it does not work suddenly I would be running the / f or change message to say to / f help. Except that in the lang.yml we do not see the variable that refers to the / f, here the line 
Toomany:  b> Strange argument "p% 1 $ s <b".< I Use the command like this: ' 

Thank you for your help :)

**Comment by drtshock** [comment]
Can anyone translate?

**Comment by SupaHam** [comment]
Hello, I use your FactionUUID plugin but I am having a problem using it. In the config.yml I edited the `/f help` settings for custom help output. However, after this change, I am no longer able to execute `/f`, but `/f help` functions. The problem is that when one uses the wrong command in game, it outputs "You got the wrong order" and to execute `/f` instead.

I tried to change it so that the message would guide the command sender to `/f help` instead of `/f`, but as it turns out the lang.yml message node does not have a variable for that output. Here's the message `Toomany: b> Strange argument "p% 1 $ s <b".< I Use the command like this:`

---

In short, I believe he's trying to change the message output so that it tells users to try using '/f help' instead of '/f' when too many arguments are supplied or something

**Comment by SayR0** [comment]
yes thank you to help me to translate this message i'm french and didn't write very well english =)
